README.md

```diff
@@ -99,6 +99,24 @@ If the toolbar `<script>` is accidentally included on your site, and `SentryTool
 
 #### Conditionally inserting script tag
 
-It's possible to dynamically insert the script tag inside an SPA app, prior to calling `SentryToolbar.init()`, so that only users who are elegible . See docs/conditional-script.md for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login
+It's possible to dynamically insert the script tag inside an SPA app, prior to calling `SentryToolbar.init()`, so that only users who are eligible . See docs/conditional-script.md for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login
 
 This example code will be eventually implemented as an NPM package, but for now it's something to be done manually.
+
+### Injecting the toolbar into a site
+
+To inject the toolbar into 'random' sites, configure and paste the following snippets into the console:
+
+1.
+```javascript
+var s = document.createElement('script');
+s.src = "https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js";
+document.head.appendChild(s);
+```
+2.
+```javascript
+window.SentryToolbar.init({ ... });
+```
+
+To view feature flags, you will need to set your feature flag configuration before you initialize the toolbar.
+Also remember to edit the Project Settings > Developer Toolbar in sentry.io to allow the domain that the webapp uses!
```

Comment:
Do these both mean the same thing? We could probably clarify and consolidate the message.